report/report.tex

```diff
@@ -128,7 +128,12 @@ \section{Kontext}
 den Schwerpunkt des Gameplays stärker auf die Vorausplanung und langfristigen Strategien
 zu legen und reaktives Gameplay in den Hintergrund zu stellen.
 
-% (Eventuell ein 'game play' Bild reinpasten)
+\begin{figure}[h]
+  \caption{Darwin Gameplay}
+  \centering
+  \includegraphics[width=0.95\textwidth]{./img/darwin-gameplay}
```

Comment:
Everything except `.tex` is currently in the `.gitignore` blacklist - could you tune the `.gitignore` ruleset and commit the image as well?

Reply:
of course, totally forgot that